app/assets/javascripts/discourse/app/routes/application.js

```diff
@@ -47,13 +47,6 @@ const ApplicationRoute = DiscourseRoute.extend(OpenComposer, {
       });
     },
 
-    toggleSidebar() {
-      // enables CSS transitions, but not on did-insert
-      document.querySelector("body").classList.add("sidebar-animate");
-
-      this.controllerFor("application").toggleProperty("showSidebar");
-    },
```

Comment:
This deals with a controller state so we can just have the action inside the controller.